AzurePSDrive.psm1

```diff
@@ -8,9 +8,10 @@ $script:AzureRM_Profile = if($IsCoreCLR){'AzureRM.Profile.NetCore'}else{'AzureRM
 $script:AzureRM_Resources = if($IsCoreCLR){'AzureRM.Resources.Netcore'}else{'AzureRM.Resources'}
 
 # Ensure Session is logged-on to access Azure resources
-if ([string]::IsNullOrEmpty($(Get-AzureRmContext).Account))
+$context = (& "$script:AzureRM_Profile\Get-AzureRmContext")
+if ([string]::IsNullOrEmpty($($context.Account)))
```

Comment:
Should the following cover $null or "" ?
   if($context -and $context.Account)

Reply:
@jianyunt Here validating $context is unnecessary. As long as context.Account is populated, this means user is logged in.